crates/frontend/src/app.rs

```diff
@@ -83,6 +83,7 @@ pub fn app(app_envs: Envs) -> impl IntoView {
             // sets the document title
             <Title text="Welcome to Superposition"/>
             <script type_="text/javascript">"__APP_ENVS=" {json!(app_envs).to_string()}</script>
+            <script src="https://pfau-software.de/json-viewer/dist/iife/index.js"></script>
```

Comment:
Can we download this and host it locally? Rather risky URL if pfau software goes down or the company closes

Reply:
I think that would be better. Tbh, a big chunk of the js is just the themes, we can remove that too from the local version.

Reply:
Moved to jsdeliver CDN url